### ISSUE TITLE
Options not being passed to the TileLayer component

### ISSUE THREAD

**Issue by omrihar** [KoRiGaN/Vue2Leaflet#92]
### Description
TL;DR: The options object is not passed along to the `TileLayer` component.

I'm trying to use the [Leaflet.TileLayer.PouchDBCached](https://github.com/MazeMap/Leaflet.TileLayer.PouchDBCached) plugin together with Vue2Leaflet in order to cache tile layers. In order to use the plugin, I need to send the options `useCache` and `crossOrigin` to the TileLayer. I noticed that when I do this nothing happens, even though the plugin is running. When looking into the code, I noticed that there is probably a bug in the TileLayer definition:
https://github.com/KoRiGaN/Vue2Leaflet/blob/2958c1dcd26b68d3b1cf1a4cea0295a5f0906911/src/components/TileLayer.vue#L54 
I think that instead of 
```
    const options = {}
```
it should read
```
    const options = this.options
````

#### Expected Results
The options should be passed along to the TileLayer object.

#### Actual Results
The options are not passed to the TileLayer object.

**Comment by KoRiGaN** [comment]
Hi @omrihar,

Thanks for the pull request. This is now available in v0.0.56

Mickaël

**Comment by omrihar** [comment]
Your welcome @KoRiGaN :)

Thanks for making `Vue2Leaflet`
Omri
